--- dovecot-1.2-sieve-0.1.8/src/testsuite/Makefile.am~	2009-03-05 20:37:50.000000000 +0100
+++ dovecot-1.2-sieve-0.1.8/src/testsuite/Makefile.am	2009-07-13 11:46:11.142752409 +0200
@@ -14,8 +14,6 @@
 libs = \
 	$(top_srcdir)/src/lib-sieve/libsieve.la \
 	$(top_srcdir)/src/lib-sieve-tool/libsieve-tool.la \
-	$(dovecot_incdir)/src/lib-storage/list/libstorage_list.a \
-	$(dovecot_incdir)/src/lib-storage/register/libstorage-register.a \
 	$(STORAGE_LIBS)
 
 ldadd = \
